ngx-tour-ng-bootstrap/src/tour-anchor.directive.ts

```diff
@@ -1,14 +1,34 @@
-import { Directive, ElementRef, Host, HostBinding, Input, OnDestroy, OnInit } from '@angular/core';
-import { NgbPopover, Placement } from '@ng-bootstrap/ng-bootstrap';
+import { Directive, ElementRef, Host, HostBinding, Input, OnDestroy, OnInit, ComponentFactoryResolver, Injector, NgZone, Renderer2, ViewContainerRef } from '@angular/core';
+import { NgbPopover, Placement, NgbPopoverConfig } from '@ng-bootstrap/ng-bootstrap';
 import { TourAnchorDirective } from 'ngx-tour-core';
 import withinviewport from 'withinviewport';
 
 import { NgbTourService } from './ng-bootstrap-tour.service';
 import { INgbStepOption } from './step-option.interface';
 import { TourStepTemplateService } from './tour-step-template.service';
 
+
 @Directive({ selector: '[tourAnchor]' })
-export class TourAnchorNgBootstrapPopoverDirective extends NgbPopover {}
+export class TourAnchorNgBootstrapPopoverDirective extends NgbPopover {
+
+  private elementRef: ElementRef;
+
+  constructor(_elementRef: ElementRef, _renderer: Renderer2, injector: Injector, componentFactoryResolver: ComponentFactoryResolver,
+  viewContainerRef: ViewContainerRef, config: NgbPopoverConfig, ngZone: NgZone) {
+    super(_elementRef, _renderer, injector, componentFactoryResolver, viewContainerRef, config, ngZone);
```

Comment:
I really don't want to put any extension logic inside the `TourAnchorNgBootstrapPopoverDirective`.  Is there any way for you to accomplish the same thing in the `TourAnchorNgBootstrapDirective` by referencing `this.popoverDirective`?

That way I won't have to change this implementation every time the `NgbPopover` changes the constructor signature.